apis/nucleus/src/components/listbox/default-properties.js

```diff
@@ -1,7 +1,32 @@
+/**
```

Comment:
This file should be used by the listbox right?

Reply:
is is used by listbox code in nebula.js in useOnTheFlyModel.jsx and filterpane-handler.js